### PR TITLE
Create reverse_doubly_linked_list.java

### LINKED ISSUE #349: Reversal of Doubly link list
# Description
I would like to add a program to reverse Doubly link list in c language

# Additional context
Program would perform the reversal operation on the Doubly link list given by the user

# Important Checklist 
-   [x] I have read the contribution guidelines before making this issue.
-   [x] I have verified this issue program is not already implemented
-   [x] I have not pushed this code anywhere else (even my repo)

# Languages and Assignees:
-   [x] C:  @Anupam-Panwar
-   [x] Java: @manishita24 c

### PR DESCRIPTION
**Pull request by manishita24**
# Description

I added a program to reverse Doubly link list in java language

Fixes #349

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] I have read the contribution guidelines before making this PR.
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
+public class Main {
```
Comment (kaustubhgupta) [inline Main.java]: update java folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]
Points to be changed
- rename file to reverseDoublyLinkedList.java
- update readme

Advice(just a tip for style improvement)
- you can override toString() function in java instead of explicitly writting display function.

**Comment by manishita24** [comment]
sheesh, I am bad at this. 😭😭😭 I hope it is fine now

**Comment by kaustubhgupta** [comment]
@manishita24 Ahh code is perfectly fine now but you need to update the Java readme file too :-) 
Just update that one with your entry and this PR will be merged

**Comment by manishita24** [comment]
Done. Thank you for your patience and help.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@manishita24 Perfect thanks for contributing here! 😋

**Pull request merged**